api/internal/localizer/localizer_test.go

```diff
@@ -1461,6 +1461,62 @@ func TestCopyChartHomeError(t *testing.T) {
 	}
 }
 
+func TestLocalizeGeneratorsHelm(t *testing.T) {
+	files := map[string]string{
+		"kustomization.yaml": `generators:
+- default.yaml
```

Comment:
Does `localize` currently handle generators defined inline? This is not required for alpha imo, but if it already handled, can we add a test? If not, can we add a TODO?

Reply:
Done. Yes, we handle inline generators and I added a test.